.github/workflows/ci.yml

```diff
@@ -17,10 +17,10 @@ jobs:
       - name: Build with Gradle
         run: ./gradlew build
       - name: Publish
-        if: ${{ github.ref == 'refs/heads/main' || startswith(github.ref, 'refs/tags/') }}
+        if: ${{ github.ref == 'refs/heads/main' || startswith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/publish-as-contentgrid' }}
```

Comment:
Presumably this will be removed before merge?

Reply:
was a separate commit for a test-publication - already removed again